### DISCUSSION TITLE
改词典字体后重启失效

### DISCUSSION THREAD

**Discussion by yshumy** [xiaoyifang/goldendict-ng#925] [Q&A]
版本：6.4.3-GoldenDict_macos-12_homebrew_20230702
改词典字体后重启失效，自动恢复原来字体。
是不是 bug ？

**Comment by xiaoyifang** [comment]
![image](https://github.com/xiaoyifang/goldendict-ng/assets/105986/e5c15f4b-f550-4dc2-9871-8da0834cf166)

这部分的逻辑已经重构的了，试下新版本的。
